config/checkstyle/checkstyle.xml

```diff
@@ -70,7 +70,9 @@
         </module>
         <module name="AvoidStarImport">
```

Comment:
You are right. Using wildcard is not a bad thing always but it's also not good for everything and we can decide if it is good or bad in PR's. I can remove it, wdyt @heshamMassoud?

Reply:
@lojzatran @ahmetoz I guess I am still more convinced with the advantages of **not** using wild card imports. Regardless of this fact, this is a convention we have been taking in this lib since the start. So if we want to start changing this convention, it should be on another PR but definitely not this one 😉